NEWS.md

```diff
@@ -3,6 +3,7 @@ ClimaLand.jl Release Notes
 
 main
 --------
+-  Add profiling of soil/anopy model to buildkite pipeline PR [#561](https://github.com/CliMA/ClimaLand.jl/pull/561)
```

Comment:
Are there more news entries to be added here?

Reply:
Ill note the other changes